### ISSUE TITLE
Adding pay_transfer table

### ISSUE THREAD

**Issue by ykamin-booth** [pay-rails/pay#711]
I'm finding more and more that we want to store transfer data for Stripe Connect in our database rather than just in Stripe. Has there been any discussion on adding a pay_transfer table? This is especially the case when we have a non-1:1 ratio of charges to a transfer or vice versa so just adding extra columns to a pay_charge wasn't cutting it.

**Comment by excid3** [comment]
Yeah, I can definitely see adding this for Connect. I don't know if it'd be easy to make as an optional table since anyone not using Connect wouldn't need it.

I believe Balance Transactions include Transfers, so maybe that would be a better one to sync since it could be useful in more situations, especially if it also includes the Transfer details.

**Comment by ykamin-booth** [comment]
It does include transfers, though a whole lot else as well so it may be more complicated to sync.

**Comment by excid3** [comment]
It shouldn't be any more complicated. The only problem would be if they didn't contain the same data. If we're going to sync anything, the BalanceTransactions, while more information, would be much more valuable. It's basically the ledger of your account, which is nice.
